Sources/MapboxMaps/Viewport/States/FollowPuck/FollowPuckViewportStateOptions.swift

```diff
@@ -9,7 +9,7 @@
                 zoom: CGFloat? = 16.35,
                 bearing: FollowPuckViewportStateBearing? = .heading,
                 pitch: CGFloat? = 45,
-                animationDuration: TimeInterval = 1) {
+                animationDuration: TimeInterval = 0.3) {
```

Comment:
@pengdev okay with you if we change this default value to 0.3? When set to 1 second, I'm still seeing a pretty noticeable jump at the end of the DefaultViewportTransition.

Reply:
I think that depends on the moving speed of the puck, right? if the puck is not moving fast, this might not be a concern? Anyway I think we have provided users the flexibility to adjust it based on their use case, wdyt?

Reply:
That's true. It does depend on a lot of factors. I can revert it to 1.